cea/default.config

```diff
@@ -146,6 +146,10 @@ use-dynamic-infiltration-calculation = false
 use-dynamic-infiltration-calculation.type = BooleanParameter
 use-dynamic-infiltration-calculation.help = Use the dynamic infiltration calculation (slower!)
 
+use-daysim-radiation = true
+use-daysim-radiation.type = BooleanParameter
+use-daysim-radiation.help = set to false to use ArcGIS radiation as input, default is true
```

Comment:
Check the ArcGIS interface for the demand tool to see if the help message makes sense. Maybe it needs some reformulation?

Reply:
yep